src/dotnet/APIView/ClientSPA/src/app/_components/code-panel/code-panel.component.ts

```diff
@@ -619,6 +636,97 @@ export class CodePanelComponent implements OnChanges{
     }
   }
 
+  private searchCodePanelRowData(searchText: string) {
+    this.searchMatchedRowInfo.clear();
+    if (!searchText || searchText.length === 0) {
+      this.clearSearchMatchHighlights();
+      return;
+    }
+
+    let totalMatches = 0;
+    let matchedNodeIds = new Set<string>();
+    this.codePanelRowData.forEach((row) => {
```

Comment:
So search seems to work only for code tokens. Is there a way to search for a review comment? For e.g. if architects want to search about a specific comment or a text in a comment they left earlier. or it could be a search for a text in system comment? I assume this search feature won't cover them. right?

Reply:
This change does not include search for text in specific comment.